### DISCUSSION TITLE
Custom plugins for Liquid tags not being recognised

### DISCUSSION THREAD

**Discussion by angadhn** [mmistakes/minimal-mistakes#3960] [General]
I am trying to get a set of Liquid tags working with markdown files in `_posts/` and `_pages` via custom plugins by placing a `.rb` file in the `_plugins` directory, as described [here(https://jekyllrb.com/docs/plugins/tags/). However, I get an `unkown_tag: Liquid syntax error` with such plugins.

I imagine that I am missing something obvious here so would appreciate any help/directions in getting these custom tags to work.

**Comment by iBug** [comment]
There's nothing we can help unless you show your code.
